src/Handlers/BaseHandler.php

```diff
@@ -223,6 +236,9 @@ private function prepareCompileAndRunScript(): string
         END;
 	}
 
+	/**
+	 * During preparation creates new file with new file system.
```

Comment:
change: "Encode the code and the input to pass it to the docker container as parameters."